Backend/utils/MentiiAuth.py

```diff
@@ -0,0 +1,83 @@
+from flask import g
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+from boto3.dynamodb.conditions import Key, Attr
+import hashlib
+import ConfigParser as cp
+
+class MentiiAuthentication:
+
+  def __init__(self):
+    return
+
+  @staticmethod
+  def build_user_object(userData):
+    '''
+    Builds a standard user object to return to the client for all authenticated calls
+    '''
+    #TODO: Eventually add role to this packet
+    return {'email': userData['email']}
+
+  @staticmethod
+  def get_app_secret():
+    '''
+    Reads from the prodConfig.ini file for the appSecret
+    '''
+    parser = cp.ConfigParser()
+    parser.read("/config/prodConfig.ini")
+    return parser.get('MentiiAuthentication', 'appSecret')
+
+  @staticmethod
+  def generate_auth_token(userCredentials, expiration = 86400):
+    '''
+    Generates the auth token based off of the user's email and password with a default expiration duration.
+    Default token will last 1 day.
+    '''
+    appSecret = MentiiAuthentication.get_app_secret()
+    s = Serializer(appSecret, expires_in = expiration)
+    email = userCredentials['email']
+    password = userCredentials['password']
```

Comment:
Again, what if userCredentials doesn't have email or password as a key? Added error handling or checking of keys